lilach-client/src/main/resources/org/cshaifasweng/winter/order_completed.fxml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.text.*?>
+
+<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" prefHeight="500.0" prefWidth="904.0" xmlns="http://javafx.com/javafx/10.0.2-internal" xmlns:fx="http://javafx.com/fxml/1" fx:controller="org.cshaifasweng.winter.OrderController">
+    <children>
+        <Label layoutX="46.0" layoutY="14.0" prefHeight="35.0" prefWidth="189.0" text="Thank You !">
```

Comment:
Remove the space before the exclamation mark.

Reply:
ok